R/utils_dataprocess_plots.R

```diff
@@ -50,29 +49,34 @@
         input$is_censored = factor(input$is_censored, 
                                    levels = c("FALSE", "TRUE"))
     }
+    print(input)
+    print(unique(input$predicted))
+    write.csv(input, file = "your_file_name.csv", row.names = T)
```

Comment:
what's this?

Reply:
Commit [remove csv debug](https://github.com/Vitek-Lab/MSstats/pull/107/commits/aea147a3323b1e7885865d54107f00c2fc716205)

Was written to understand the data table